cuda_phase-retrieval/run.py

```diff
@@ -9,45 +9,49 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--image", help="Image path")
-parser.add_argument("--step", help="number of iterations (default 20)", type=int, default=20)
-parser.add_argument("--beta", help="beta (default 0.8)", type=float, default=0.8)
-parser.add_argument("--type", help="Hybrid, Input-Ouput, or Output-Output", default="hybrid")
+parser.add_argument("--mask", help="Mask value (default 1). This will automatically create array of the specified value with the same size of the input image", type=float, default=1)
+parser.add_argument("--step", help="Number of iterations (default 20)", type=int, default=20)
+parser.add_argument("--beta", help="Beta (default 0.8)", type=float, default=0.8)
+parser.add_argument("--mode", help="Hybrid, Input-Ouput, or Output-Output", default="hybrid")
+parser.add_argument("--type", help="Language used to run phase retrieval. CUDA (default) or Python. ", default="cuda")
```

Comment:
Mode `Python` does not work:

```bash
$ python run.py --image example_images/a.png  --type Python
Running phase retrieval...
Traceback (most recent call last):
  File "run.py", line 41, in <module>
    result = phase_retrieval_python.fienup_phase_retrieval(image, mask, mode, type, beta, array_random)
  File "/home/sehrig/project/student_project_python_bindings/cuda_phase-retrieval/build/phase_retrieval_python.py", line 37, in fienup_phase_retrieval
    assert steps > 0, 'steps must be a positive number'
TypeError: '>' not supported between instances of 'str' and 'int'
```

Reply:
yes I forgot to change the parameters in the python part